lib/error.js

```diff
@@ -13,6 +13,12 @@ function FormError(key, options, req, res) {
             return options.message || this.getMessage(key, options, req, res);
         }
     });
+    Object.defineProperty(this, 'title', {
+        enumerable: true,
+        get: function () {
+            return options.title || 'Oops, something went wrong';
```

Comment:
Ah yeah - I hadn't opened the file.  For the sake of consistency I guess it would be nicer to go with `|| getTitle()` - especially as it would give control over the magic string